### DISCUSSION TITLE
Welcome to Sweet-Pop Discussions!

### DISCUSSION THREAD

**Discussion by PROxZIMA** [PROxZIMA/Sweet-Pop#16] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by daniloogoncalves** [comment]
Hi,
First of all, this is an excellent project!
I did the installation of CSS file, but my customization was not the same as the project. The colored line is not going under the URL Bar and the window buttons as close and zoom are in the middle bar.
Is there a way to fix that? I'm using Firefox 100.

<img width="1112" alt="firefox-image" src="https://user-images.githubusercontent.com/72620526/166612619-ad27e088-c58b-437b-93d0-722fd12a5d08.png">

![firefox-image2](https://user-images.githubusercontent.com/72620526/166614971-846f890c-7113-4f1a-b06b-0cedf03cdf90.png)

**Reply by PROxZIMA** [reply]
1) I have not tested the theme on Firefox 100 yet. I'll test it in a day or two.
2) Window control button are affected due to the OS placement type (left/right).
- Left placement 
![image](https://user-images.githubusercontent.com/43103163/166703370-287a0548-d1a3-456c-b02f-9668aacd893e.png)
3) Try to reduce the button count in url bar. For example adding them to the overflow menu. 
![image](https://user-images.githubusercontent.com/43103163/166704160-f44d4c60-360e-4d91-9af1-3fac9c9052ad.png)

**Comment by daniloogoncalves** [comment]
Hi!

I also tested the theme at Firefox 99 and 98, and both had the same issue.

Thanks


De: Pratik Pingale ***@***.***>
Data: quarta-feira, 4 de maio de 2022 11:35
Para: PROxZIMA/Sweet-Pop ***@***.***>
Cc: daniloogoncalves ***@***.***>, Comment ***@***.***>
Assunto: Re: [PROxZIMA/Sweet-Pop] Welcome to Sweet-Pop Discussions! (Discussion #16)

  1.  I have not tested the theme on Firefox 100 yet. I'll test it in a day or two.
  2.  Window control button are affected due to the OS placement type (left/right).

  *   Left placement
[image]<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F43103163%2F166703370-287a0548-d1a3-456c-b02f-9668aacd893e.png&data=05%7C01%7C%7C657a556fae6d4096c00e08da2ddb4df8%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637872717198517349%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=ZUGTm7ZbyC7rMQ9mgrcSbtZttgm6edTgmCsYLTYM5C0%3D&reserved=0>

  1.  Try to reduce the button count in url bar. For example adding them to the overflow menu.
[image]<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F43103163%2F166704160-f44d4c60-360e-4d91-9af1-3fac9c9052ad.png&data=05%7C01%7C%7C657a556fae6d4096c00e08da2ddb4df8%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637872717198517349%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=h5XL54S%2BM9T%2FCYjIwXrs3zJ2jWp800meQqiJNzHur8E%3D&reserved=0>

**Comment by daniloogoncalves** [comment]
Hi there!
I was able to correct issues 2, 3 and 4.
Do you have any idea how to fix issue 1?
Thanks in adv.

![firefox-corrigir](https://user-images.githubusercontent.com/72620526/166975800-6113d2e2-a1f8-4cf0-8e61-415c75d9cc51.png)

**Reply by PROxZIMA** [reply]
I'll boot up a Mac iso and test the theme because I believe this issue is specially related to Mac and is caused by some CSS properties of

1) `nav-bar`
2) window control buttons (`titlebar-buttonbox-container`)
3) `new-tab-button`
4) `titlebar-spacer[post-tab]`

I would appreciate if you post the changes you made to correct issues 2, 3 and 4.

**Comment by PROxZIMA** [comment, marked as answer]
Hey!! This issue is solved in #19 🎉🎉
You can look at the latest changes and pull them

**Reply by daniloogoncalves** [reply]
Perfect! It's working. I only include a code below for remove buttonbox.

.titlebar-buttonbox-container { display: none; }

**Reply by daniloogoncalves** [reply]
Another thing. Is it possible to change the order of the back and forward buttons? It is more intuitive to go back to the left.

<img width="239" alt="Screenshot 000322" src="https://user-images.githubusercontent.com/72620526/167202566-2e189c8a-d374-49b9-94db-32b60fff7bf8.png">

**Reply by PROxZIMA** [reply]
Is there some kind of right to left (`rtl`) configuration enabled on your system?
For me its normal 
![a](https://user-images.githubusercontent.com/43103163/167247306-a7aaf009-c564-4abf-914f-c0b3c7ba371a.png)

**Reply by daniloogoncalves** [reply]
I don't know, where can I check this configuration?

**Comment by daniloogoncalves** [comment]
And, how I could remove this bar on the navbar. It appears when I'll type something.


![navbar-firefox](https://user-images.githubusercontent.com/72620526/167254854-5e8dff0a-2341-4b00-823c-bfbc36dce576.png)

**Reply by PROxZIMA** [reply]
This can be due to `.urlbarView-results` padding. Check `.urlbarView-body-inner` and `.urlbarView-body-outer` padding too. This might need some debugging from you end :/
![a](https://user-images.githubusercontent.com/43103163/167258814-d2b308cf-6cee-4418-aacf-31cf2d5e0b8c.png)

**Comment by daniloogoncalves** [comment]
Hi,
My browser is on version 104.0.1 and it's showing two lines.. one from the tab and one from the bookmarks... how can I remove the line from the bookmarks?

![Screenshot 000800](https://user-images.githubusercontent.com/72620526/188332567-61abc4f6-8803-4420-b85f-8d3da335bd60.png)

**Reply by PROxZIMA** [reply]
Fixed in #26. Actually I was planning to fix this but had no idea how to do. The fix just clicked in. lol

**Reply by daniloogoncalves** [reply]
Fixed! Tks

**Comment by daniloogoncalves** [comment]
I don't know. But how could I be able to change?



De: Pratik Pingale ***@***.***>
Data: sábado, 7 de maio de 2022 06:06
Para: PROxZIMA/Sweet-Pop ***@***.***>
Cc: daniloogoncalves ***@***.***>, Comment ***@***.***>
Assunto: Re: [PROxZIMA/Sweet-Pop] Welcome to Sweet-Pop Discussions! (Discussion #16)

Is some right to left (rtl) configuration enabled on your system?
For me its normal
[a]<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F43103163%2F167247306-a7aaf009-c564-4abf-914f-c0b3c7ba371a.png&data=05%7C01%7C%7C52056262fc9347060e5408da3008ee2c%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637875112167181515%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=ANdRqdHxHhPevKwiMo295QlICqOJqv6HnRDpPAdCIos%3D&reserved=0>

**Reply by PROxZIMA** [reply]
I'm afraid, I can't help in this case. I have used MacOS only once and that to for installation testing. I'm not well versed with its settings. Also I didn't face this issue when I was testing.

**Reply by daniloogoncalves** [reply]
No worries, I reinstalled Firefox and the issue was fixed. Tks
